### PR TITLE
Update Awake documentation with --expire-at command-line

### PR DESCRIPTION
**Pull request by rpr69**

### CODE REVIEW

File: hub/powertoys/awake.md [hunk 1]
```diff
@@ -68,3 +69,8 @@ When setting the value for the `--time-limit` parameter, both of these formats w
 
 - PowerToys.Awake.exe --time-limit 36000
 - PowerToys.Awake.exe --time-limit=36000
+
+When setting the value for the `--expire-at` parameter, these formats will be accepted based on [DateTime.Parse](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.parse?view=net-7.0#examples)
```
Comment (Jak-MS) [inline hub/powertoys/awake.md]:
```suggestion
When setting the value for the `--expire-at` parameter, these formats will be accepted based on [DateTime.Parse](/dotnet/api/system.datetime.parse?view=net-7.0#examples)
```

**Comment by Jak-MS** [comment]
@mattwojo 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.

NOTE: The errors in the link validation will need to be addressed before this can be merged. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: hub/powertoys/awake.md [hunk 2]
```diff
@@ -68,3 +69,8 @@ When setting the value for the `--time-limit` parameter, both of these formats w
 
 - PowerToys.Awake.exe --time-limit 36000
 - PowerToys.Awake.exe --time-limit=36000
+
+When setting the value for the `--expire-at` parameter, these formats will be accepted based on [DateTime.Parse](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.parse)
```
Comment (crutkas) [inline hub/powertoys/awake.md]: can we remove the en-us so it is language agnostic.

```suggestion
When setting the value for the `--expire-at` parameter, these formats will be accepted based on [DateTime.Parse](https://learn.microsoft.com/dotnet/api/system.datetime.parse)
```

**Review by crutkas — APPROVED** [review]

**Comment by crutkas** [comment]
@dend any opposition for doc updates?

File: hub/powertoys/awake.md (hunk 2 above)
Comment (dend) [reply hub/powertoys/awake.md]: Indeed, this needs to be locale-agnostic.

File: hub/powertoys/awake.md [hunk 3]
```diff
@@ -68,3 +69,8 @@ When setting the value for the `--time-limit` parameter, both of these formats w
...
+
+- PowerToys.Awake.exe --expire-at=17:00:00 will expire at 5PM of the current day
```
Comment (dend) [inline hub/powertoys/awake.md]: This is more of a general comment, but when we document commands, we need to make sure that we wrap them in single quotes, to separate from the explanatory text. Right now it all blends together.

File: hub/powertoys/awake.md [hunk 4]
```diff
@@ -57,6 +57,7 @@ PowerToys Awake can also be executed as a standalone application, directly from
 | `--use-pt-config` | Use the PowerToys configuration file to manage the settings. This assumes that there is a `settings.json` file for Awake, generated by PowerToys, that contains all required runtime information. This includes the Behavior Mode (indefinite or timed), whether screens should be kept on, and what the values for hours and minutes are for a temporary keep-awake.<br/>When this argument is used, all other arguments are ignored. Awake will look for changes in the `settings.json` file to update its state. |
 | `--display-on`    | Determines whether the screens should be kept on or off while the machine is kept awake. Expected values are `true` or `false`. |
 | `--time-limit`    | Duration, in seconds, during which Awake keeps the computer awake. Can be used in combination with `--display-on`. |
+| `--expire-at`     | Expiration date and/or time during which Awake keeps the computer awake. Can be used in combination with `--display-on`. |
```
Comment (dend) [inline hub/powertoys/awake.md]: ...date/or time _when_ Awake will turn off and resume the standard power state.

**Review by dend — CHANGES_REQUESTED** [review]

**Comment by dend** [comment]
Requested some changes, but once these are in - I am good with the updates.

@crutkas

File: hub/powertoys/awake.md (hunk 2 above)
Comment (mattwojo) [reply hub/powertoys/awake.md]: Done - we want all links inside learn.ms to be relative links as well.

**Comment by mattwojo** [comment]
Thanks for contributing this @rpr69! Merged.

**Pull request merged**